### DISCUSSION TITLE
For organizations that need help

### DISCUSSION THREAD

**Discussion by miripiruni** [gemini-testing/testplane#838] [Announcements]
If your company tries to migrate to Testplane but you have some blockers, add a post here and reference the issues that you have created. We will try to prioritize bug fixes/feature requests that are coming from big teams/companies.

The post should contain the name of the company and the size of the team.

Feel free to use this discussion also if you already migrated to Testplane but really need help with some issues.

**Comment by Dannymex420** [comment]
termux
